scripts/functions_deploy.py

```diff
@@ -57,6 +57,7 @@ def main(args):
     parser.add_argument('-c', '--common_configFilePaths', help="configuaration file", action='append')
     parser.add_argument('--common_functions', required=False, help="directory where the cloud functions are located")
     parser.add_argument('--cloudfunctions_namespace', required=False, help="cloud functions namespace")
+    parser.add_argument('--cloudfunctions_apikey', required=False, help="cloud functions apikey")
```

Comment:
This changed code is same as in `functions_test.py`, consider to move it into wawCommons.

Reply:
Functions which handled this was created